SoftLayer/CLI/virt/create.py

```diff
@@ -121,10 +121,7 @@ def _parse_create_args(client, args):
     # Get the SSH keys
     if args.get('key'):
         keys = []
-        for key in args.get('key'):
```

Comment:
Was there any reason for moving these lines to their own function?

Reply:
The problem was that it exceeded the maximum number of branches in the method and I only made a refactor of the for cycle in another method using the IDE's refactor options.